cmd/sidetree-server/main.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/go-openapi/loads"
+	"github.com/trustbloc/sidetree-node/restapi"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+
+	"net/http"
+	"strings"
+
+	"github.com/go-openapi/runtime/middleware"
+
+	"github.com/go-openapi/errors"
+
+	"github.com/trustbloc/sidetree-core-go/pkg/batch"
+	"github.com/trustbloc/sidetree-core-go/pkg/dochandler"
+	"github.com/trustbloc/sidetree-core-go/pkg/dochandler/didvalidator"
+	"github.com/trustbloc/sidetree-core-go/pkg/processor"
+
+	"github.com/go-openapi/runtime"
+	"github.com/trustbloc/sidetree-fabric/pkg/context"
+	"github.com/trustbloc/sidetree-node/pkg/requesthandler"
+	"github.com/trustbloc/sidetree-node/restapi/operations"
+)
+
+const didDocNamespace = "did:sidetree:"
```

Comment:
Would this did method name("did:sidetree:") be constant for all instances/deployments of sidetree solutions ?

Reply:
In protocol spec they use only this method/namespace name. Reference node implementation
has it is configured in yaml file.